core/templates/dev/head/pages/admin/admin.html

```diff
@@ -125,7 +125,10 @@
     <script src="{{TEMPLATE_DIR_PREFIX}}/expressions/ExpressionEvaluatorService.js"></script>
     <script src="{{TEMPLATE_DIR_PREFIX}}/expressions/ExpressionParserService.js"></script>
     <script src="{{TEMPLATE_DIR_PREFIX}}/domain/utilities/UrlInterpolationService.js"></script>
-
+    <script src="{{TEMPLATE_DIR_PREFIX}}/pages/moderator/Moderator.js"></script>
```

Comment:
Is this needed here (and also in `exploration_editor.html`,  `exploration_player.html`)?